fusor-ember-cli/app/components/text-f.js

```diff
@@ -40,6 +40,12 @@ export default Ember.Component.extend({
     }
   }),
 
+  invalidNetworkRange: Ember.computed('value', 'isNetworkRange', function() {
+      // TODO
+      return false;
```

Comment:
@jmontleon, merging this. I will create a separate PR for this validation and also for Matt Reid's addition to the wire frames for option to "Create an administrative password for the overcloud."